plugins/reolink/src/reolink-api.ts

```diff
@@ -431,6 +431,59 @@ export class ReolinkCameraClient {
         };
     }
 
+    async setWhiteLedState(on?: boolean, brightness?: number) {
+        const url = new URL(`http://${this.host}/api.cgi`);
+
+        const settings: any = { channel: this.channelId };
+
+        if (on !== undefined) {
+            settings.state = on ? 1 : 0;
+        }
+
+        if (brightness !== undefined) {
+            settings.bright = brightness;
+        }
+
+        const body = [{
+            cmd: 'SetWhiteLed',
+            param: { WhiteLed: settings }
+        }];
+
+        const response = await this.requestWithLogin({
+            url,
+            method: 'POST',
+            responseType: 'json',
+        }, this.createReadable(body));
+
+        const error = response.body?.[0]?.error;
+        if (error) {
+            this.console.error('error during call to setWhiteLedState', JSON.stringify(body), error);
+        }
+    }
+
+    async getWhiteLedState() {
```

Comment:
this does not look to be called, and I dont see anywhere that this value is polled to update the state from outside of scrypted?

Reply:
yes was a leftover, removed